### ISSUE TITLE
How do you actually use this?

### ISSUE THREAD

**Issue by emptyslot** [noatpad/obsidian-banners#156]
Got the thing installed but the tutorial doesnt say how to use commands. I feel stupid for not knowing but might be nice to show how to do this.

**Comment by khanhtoan529** [comment]
open command palette and search for `banner`, you will see some command there.

**Comment by orvpagadua** [comment]
YOU CANT, THIS IS ABANDONED.

**Comment by indigofairyx** [comment]
yeah as far as i can tell this doesn't work at all, should be removed from the plugins

none of these options work, nor does frontmatter


```banner
banner:![[\notes_media\_banners\SNY03336.JPG]]
```

```banner
banner:![[\\notes_media\\_banners\\SNY03336.JPG]]
```

```banner
banner:![[/notes_media/_banners/SNY03336.JPG]])
```

```banner
![[//notes_media//_banners//SNY03336.JPG]])
```

**Comment by orvpagadua** [comment]
reinstalled this using brat -- it works but comes in conflict when creating tables.

**Comment by indigofairyx** [comment]
in yaml i get...
![image](https://github.com/noatpad/obsidian-banners/assets/126439010/53d84d6c-c1fd-4760-9b3e-d01c79c85310)


in the body i get
![image](https://github.com/noatpad/obsidian-banners/assets/126439010/247ee01a-9109-4d38-b286-b1c1fbce0663)


on both regular & brat installs

**Comment by Louis-Dittmar** [comment]
I use it like this:

![image](https://github.com/user-attachments/assets/cd1cab8b-f529-4414-9c0f-95391d47b316)

Hope this works.
